server.py

```diff
@@ -185,6 +204,17 @@ def update(self):
             self.AllPlayers[bike_key].x += speed * math.sin(self.AllPlayers[bike_key].heading)
             self.AllPlayers[bike_key].z += speed * math.cos(self.AllPlayers[bike_key].heading)
 
+        if current_time - self.LastBoosters > (10 * 1000) and len(self.boosters) < 10 and self.UsersNum:
+            for i in range(min(3, 10 - len(self.boosters))):
+                r = 1000
+                self.boosters.append(Point3d(randint(-r, r), 1, randint(-r, r)))
```

Comment:
Do you spawn in 1000x1000 square? Our field is 1000x1600 ([-500, 500], [-800, 800]), but spawn in a little spaller area. (We don't want boosters too close to the wall